### PR TITLE
Tests for checking GH visibility

### PR DESCRIPTION
**Pull request by jessitron**
we should get it on the push, though.

### CODE REVIEW

File: src/atomist.config.ts [hunk 1]
```diff
@@ -26,8 +26,8 @@ export const configuration: Configuration = {
     name: pj.name,
     version: pj.version,
     teamIds: [
-         // "T1JVCMVH7",
-       "T5964N9B7",    // spring-team
+          "T1JVCMVH7",
+       //"T5964N9B7",    // spring-team
```
Comment (ddgenome) [inline src/atomist.config.ts]: Do we want this to be the default?

File: src/handlers/events/delivery/build/BuildOnScanSuccessStatus.ts [hunk 2]
```diff
@@ -19,11 +19,12 @@ import { EventFired, EventHandler, HandlerContext } from "@atomist/automation-cl
 import { GitHubRepoRef } from "@atomist/automation-client/operations/common/GitHubRepoRef";
 import { GitCommandGitProject } from "@atomist/automation-client/project/git/GitCommandGitProject";
 import { ProjectListenerInvocation } from "../../../../common/listener/Listener";
-import { currentPhaseIsStillPending, GitHubStatusAndFriends, nothingFailed, Phases, previousPhaseSucceeded } from "../../../../common/phases/Phases";
+import { currentPhaseIsStillPending, GitHubStatusAndFriends, nothingFailed, Phases } from "../../../../common/phases/Phases";
 import { addressChannelsFor } from "../../../../common/slack/addressChannels";
 import { Builder } from "../../../../spi/build/Builder";
 import { OnAnySuccessStatus } from "../../../../typings/types";
 import { StatusSuccessHandler } from "../../StatusSuccessHandler";
+import { previousPhaseSucceeded } from "../../../../common/phases/phaseOrdering";
```
Comment (ddgenome) [inline src/handlers/events/delivery/build/BuildOnScanSuccessStatus.ts]: This does not appear to be in order. Does this build? Oh, it looks like this repo has no Travis build.

File: src/atomist.config.ts (hunk 1 above)
Comment (jessitron) [reply src/atomist.config.ts]: nah. keeps moving. I'll make it an env var in a separate PR

Comment (johnsonr) [reply src/atomist.config.ts]: I've done this

**Comment by johnsonr** [comment]
Test looks good, conflicts need to be fixed

**Comment by jessitron** [comment]
This PR is so old it has ot be redone. I have hte etsts in a local branch, and thte restt of it in a new PR, and issue #161

**Pull request closed without merging**